tests/v2/test_wakunode.nim

```diff
@@ -187,3 +187,51 @@ procSuite "WakuNode":
       (await completionFut.withTimeout(5.seconds)) == true
     await node1.stop()
     await node2.stop()
+
+  asyncTest "Messages are correctly relayed":
```

Comment:
Just to make sure, does this test fail with the previous behaviour?

Reply:
apparently it does not. I need to investigate why.

Reply:
Did you figure this out? To make sure test works properly, it should definitely fail before this chunk was introduced.

(It might be that this requires setting up three full nodes for some reason, in which case the test case might be more involved and could be cut up into a separate issue and pr, per https://github.com/status-im/nim-waku/issues/158#issuecomment-707500402

Reply:
@oskarth I still haven't fully figured it out.  We are starting 3 nodes.

Reply:
I mean that this test should fail for the current code in master, and succeed with this code. Otherwise it doesn't test what we want.

(Out of process nodes don't seem necessary to me to reproduce behaviour, but they might be, in which case that could be a separate issue).

Reply:
@oskarth I think they are, when testing it with this code in master it seems to still pass. So I assume its an in / out of process thing, I wouldn't know what else the issue could be on why this test works in master.

Reply:
Ok, can we break that out into an issue as suggested here https://github.com/status-im/nim-waku/issues/158#issuecomment-707500402? That'd be the obvious way to improve testing, as that's a bug that bit us.

Reply:
When you ran the test on master, did you run it before this commit? https://github.com/status-im/nim-waku/commit/b2edfe8dda3e28d0e45f7ae7118c7e2def9c41af

Reply:
@oskarth yes I did